modules/howtos/examples/create-remove-bucket.cc

```diff
@@ -0,0 +1,65 @@
+#include <libcouchbase/couchbase.h>
+#include <cassert>
+#include <cstdlib>
+#include <string>
+
+static void http_callback(lcb_t, int, const lcb_RESPHTTP *resp)
+{
+    printf("Operation completed with HTTP code: %d\n", resp->htstatus);
+    printf("Payload: %.*s\n", (int)resp->nbody, (char *)resp->body);
+}
+
+int main(int, char **)
+{
+    lcb_create_st crst = {};
+    lcb_t instance;
```

Comment:
it seems like this example is not for libcouchbase3. Could you make sure it compiles and works with current version of the library?

Reply:
I've deleted this file and instead now referencing the examples from management-bucket-create.cc and management-bucket-drop.cc in the devguide/examples directory.